playbooks/service/docker.yml

```diff
@@ -13,12 +13,12 @@
     - role: debops.etc_services
       tags: [ 'role::etc_services' ]
       etc_services__dependent_list:
-        - '{{ docker_etc_services_dependent_list }}'
+        - '{{ docker__etc_services_dependent_list }}'
 
     - role: debops.ferm
       tags: [ 'role::ferm' ]
       ferm__dependent_rules:
-        - '{{ docker_ferm_dependent_rules }}'
+        - '{{ docker__ferm_dependent_rules }}'
```

Comment:
This needs to be renamed to `docker__ferm__dependent_rules` after the corresponding variable is renamed.